### PR TITLE
Update libmesh

### PR DESCRIPTION
**Pull request by lindsayad**
Get double curly braces for std::array initialization so we can pass our
minimu clang requirements.

### CODE REVIEW

**Comment by dschwen** [comment]
@lindsayad I think this will break die to the deprecated numerical `size()` method on vector values` being removed

**Comment by lindsayad** [comment]
K I'll work on getting things updated. Any broken apps can have their use
of "size" replaced with "norm" ahead of this PR getting merged

**Comment by dschwen** [comment]
I have a Marmot fix ready

**Comment by dschwen** [comment]
Something is screwy with the Magpie recipe... @milljm 
but it's not related.

https://civet.inl.gov/job/538224/

File: conda/libmesh/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set build = 0 %}
 {% set strbuild = "build_" + build|string %}
-{% set version = "2020.05.21" %}
+{% set version = "2020.06.2" %}
```
Comment (dschwen) [inline conda/libmesh/meta.yaml:3]: should this be `2020.06.02` ?

**Review by dschwen — DISMISSED** [review]
If the package version is ok with a single digit day this is good to go.

**Comment by lindsayad** [comment]
I'll make it `2020.06.02`

**Review by dschwen — DISMISSED** [review]

**Review by dschwen — DISMISSED** [review]

**Comment by lindsayad** [comment]
Hahaha you keep approving it but I keep negating it. I had the wrong LDFLAGS for stripping the symbols before. Should be good now.

@loganharbour hopefully this build looks to your satisfaction?

**Review by loganharbour — APPROVED** [review]
👍  let's hope this is finally done!

We should probably start preparing some documentation about use of `moose-libmesh` for running vs development

**Comment by dschwen** [comment]
Uh, this builds all methods now and then strips symbols? I'm sure there is some wisdom in that... debugging just moose without symbol errors?

**Comment by loganharbour** [comment]
Without libMesh symbol errors, which were already not usable.

**Pull request merged**